.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
+  "arrowParens": "avoid",
```

Comment:
Prettier 2 changed its defaults. So decided to stick with previous defaults in order to not introduce too many changes.